controllers/capa_controller.go

```diff
@@ -140,8 +140,9 @@ func (r *CAPAClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		ConfigName:       key.ConfigName(awsCluster.Name),
 		Installation:     r.Installation,
 		Region:           awsCluster.Spec.Region,
-		// This is a hack to allow CAPI clusters to drop the 'release.giantswarm.io/version' label.
-		ReleaseVersion: "20.0.0-alpha1",
+		// Change to this once we have all clusters in 25.0.0
+		// ReleaseVersion:   key.Release(cluster),
+		ReleaseVersion: "25.0.0",
```

Comment:
Couldn't we use the real value of the `release` label and `IsCAPARelease` would return `true` if release >= `25.0.0` or the label is empty?

Reply:
This is the implementation in line 143, but currently clusters in 0.76.0 for example would break because they dont have a release